Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.21-alpine AS builder
+
+WORKDIR /app
+
+RUN apk add --no-cache git make
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+RUN make build
+
+FROM alpine:latest
```

Comment:
Let's used fixed alpine versions to make sure the builder has the same as this one